src/sdk/namespace_fs.js

```diff
@@ -968,7 +969,7 @@ class NamespaceFS {
             const stat = await file.stat(fs_context);
             this._throw_if_delete_marker(stat);
             // await this._fail_if_archived_or_sparse_file(fs_context, file_path, stat);
-
+            this._handle_read_part_number(params, res, stat);
```

Comment:
OK this should also be called in read_object_md to return the correct range for a part_number on HeadObject... I think I had it in another commit that was lost into a limbo between git branches...

See 
> [partNumber](https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html#API_HeadObject_RequestSyntax)
Part number of the object being read. This is a positive integer between 1 and 10,000. Effectively performs a 'ranged' HEAD request for the part specified. Useful querying about the size of the part and the number of parts in this object.